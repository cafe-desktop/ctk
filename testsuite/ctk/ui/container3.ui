<!-- invalid packing element -->
<interface>
  <object class="CtkBox">
    <child>
      <object class="CtkLabel" id="widget1"/>
      <packing>
        <warbl/>
      </packing>
    </child>
  </object>
</interface>
