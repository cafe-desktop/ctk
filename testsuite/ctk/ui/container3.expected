ERROR: ctk-builder-error-quark 1
.:7:17 Unsupported tag for CtkContainer: <warbl>
